backend/build.gradle

```diff
@@ -178,7 +178,7 @@ sonarqube {
         property "sonar.host.url", "https://sonarcloud.io"
         // In order to get both the frontend and backend code to report, we need to set the sonar project to the root directory.
         property "sonar.sources", "backend/src/main/java,frontend/src"
-        property "sonar.exclusions", "frontend/src/**/*.test.*,frontend/src/stories/**/*,frontend/src/**/*.stories.tsx"
+        property "sonar.exclusions", "frontend/src/**/*.test.*,frontend/src/stories/**/*,frontend/src/**/*.stories.tsx", "frontend/src/lang/*.ts"
```

Comment:
Is there a reason you wanted to stop all sonar checks for the lang files rather than just the duplicate checks?

I'm not sure it'll matter either way, just don't want to disable _too_ much of our automated QA

Reply:
Nah, that was just me experimenting (and learning what `cpd` stands for). Rolled back in bbb34d8.